.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -5,25 +5,27 @@ about: Create a bug report to help us improve Streamlink
 
 <!--
 Thanks for reporting a bug!
+USE THE TEMPLATE, otherwise, your bug report may be rejected.
 
-Please read the contribution guidelines before creating a bug report:
+First, see the contribution guidelines:
+https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#contributing-to-streamlink
 
-(https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#contributing-to-streamlink)
-
-Also check the list of known issues before reporting an issue!
-
-Please fill out the following template. Be as detailed as possible.
+Also check the list of open and closed bug reports:
+https://github.com/streamlink/streamlink/issues?q=is%3Aissue+label%3A%22bug%22
 
 Please see the text preview to avoid unnecessary formatting errors.
 -->
 
+
 ## Bug Report
 
-- [ ] I have read the contribution guidelines. <!-- Replace the space with an x to check the box: [x] -->
+<!-- Replace [ ] with [x] in order to check the box -->
+- [ ] This is a bug report and I have read the contribution guidelines.
+
 
 ### Description
 
-<!-- Explain the bug as clearly as you can. How is functionality impacted? Which OS/Player/Streamlink version are you using? Etc. -->
+<!-- Explain the bug as detailed as you can. -->
```

Comment:
Grammatically this is a weird sentence, maybe something like `Explain the bug as thoroughly as you can`? Also since we're dropping the requirement of player and such does this open us up to fewer details? Granted people weren't doing it any way...